ProductWarehouse.Infrastructure/Configuration/MockyClientOptions.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ProductWarehouse.Infrastructure.Configuration;
+
+public class MockyClientOptions
+{
+    public const string MockyClient = "MockyClient";
```

Comment:
"MockyClient" if the name is the same as  string value best approach is ` nameof(MockyClient)`